src/helpers/gho-launch/tests/AaveV3GHOLaunchTest.sol

```diff
@@ -0,0 +1,407 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IPool as IPool_CCIP} from 'src/interfaces/ccip/tokenPool/IPool.sol';
+import {IClient} from 'src/interfaces/ccip/IClient.sol';
+import {IInternal} from 'src/interfaces/ccip/IInternal.sol';
+import {AaveV3GHOLaneTest} from './AaveV3GHOLaneTest.sol';
+import {GhoCCIPChains} from '../constants/GhoCCIPChains.sol';
+import {IACLManager} from 'aave-address-book/AaveV3.sol';
+import {IGhoToken} from 'src/interfaces/IGhoToken.sol';
+import {IOwnable} from 'aave-address-book/common/IOwnable.sol';
+import {IUpgradeableBurnMintTokenPool_1_5_1} from 'src/interfaces/ccip/tokenPool/IUpgradeableBurnMintTokenPool.sol';
+import {IGhoAaveSteward} from 'src/interfaces/IGhoAaveSteward.sol';
+import {IGhoCcipSteward} from 'src/interfaces/IGhoCcipSteward.sol';
+import {IPool} from 'aave-address-book/AaveV3.sol';
+
+abstract contract AaveV3GHOLaunchTest_PreExecution is AaveV3GHOLaneTest {
```

Comment:
This tests looks like mixes Aave Pool configurations with Gho lane tests. provably should be separated, as for the case of Ink, there will be no pool for starters. (Maybe later will be added to Ink whitelabel, but the first AIP is just for launching GHO, so does not make sense to test that there is a Pool configuration)